src/PageLayout/PageLayout.tsx

```diff
@@ -760,6 +765,7 @@ const Pane = React.forwardRef<HTMLDivElement, React.PropsWithChildren<PageLayout
           {...labelProp}
           {...(id && {id: paneId})}
         >
+          {heading && <Heading as={headingLevel}>{heading}</Heading>}
```

Comment:
Should we just:

`PageLayout.Pane.Heading = Heading`

Reply:
I adjusted this in my [most recent commit](https://github.com/primer/react/pull/3143/commits/21518779e9059f72aad526a58a27da5e68896ee3). There's now a new slot for a heading component that's tied to `PageLayout`. This should allow someone to use `PageLayout.PaneHeading` to place a heading component within the pane, above everything else (pending feedback).